tsconfig.json

```diff
@@ -6,6 +6,7 @@
 	"include": [
 		"src/**/*.ts",
 		"typings/**/*.d.ts",
+		"*.js",
```

Comment:
Why is this necessary?

Reply:
The TypeScript-ESLint plugin can only lint files that are part of the tsconfig.json (or any other tsconfig.something.json that we might be pointing it to), otherwise it errors with:
```
error  Parsing error: ESLint was configured to run on `<tsconfigRootDir>/index.js` using `parserOptions.project`: <tsconfigRootDir>/tsconfig.json
However, that TSConfig does not include this file. Either:
- Change ESLint's list of included files to not include this file
- Change that TSConfig to include this file
- Create a new TSConfig that includes this file and include it in your parserOptions.project
See the typescript-eslint docs for more info: https://typescript-eslint.io/linting/troubleshooting#i-get-errors-telling-me-eslint-was-configured-to-run--however-that-tsconfig-does-not--none-of-those-tsconfigs-include-this-file
```